xchange-okcoin/src/main/java/com/xeiam/xchange/okcoin/service/streaming/ChannelProvider.java

```diff
@@ -3,10 +3,17 @@
 import com.xeiam.xchange.currency.CurrencyPair;
 
 /** Returns market data channel name for given connection method and currency pair */
-interface ChannelProvider {
+public interface ChannelProvider {
```

Comment:
I think it actually should be removed and replaced by a json in the /src/main/resources folder, and I don't want to sacrifice the structure for this semi-useless class